site/content/how-to/traffic-management/response-headers.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "HTTP Response Headers"
+description: "Learn how to modify response headers for your HTTP Route using NGINX Gateway Fabric."
+weight: 700
+toc: true
+docs: ""
+---
+
+[HTTPRoute](https://gateway-api.sigs.k8s.io/api-types/httproute/) filters can modify the headers during the request-response lifecycle. [HTTP Header Modifiers](https://gateway-api.sigs.k8s.io/guides/http-header-modifier/?h=request#http-header-modifiers) can be used to add, modify or remove headers in incoming requests. We have two types of [filter](https://gateway-api.sigs.k8s.io/api-types/httproute/#filters-optional) that can be used to instruct the Gateway for desired behaviour.
+
+1. [ResponseHeaderModifier](https://gateway-api.sigs.k8s.io/guides/http-header-modifier/?h=request#http-response-header-modifier) to alter headers in response before responding to the downstream.
+
+
+In this guide we will modify the headers for HTTP responses when client requests are made. For an introduction to exposing your application, we recommend that you follow the [basic guide]({{< relref "/how-to/traffic-management/routing-traffic-to-your-app.md" >}}) first.
+
+
+## Prerequisites
+
+- [Install]({{< relref "/installation/" >}}) NGINX Gateway Fabric.
+- [Expose NGINX Gateway Fabric]({{< relref "installation/expose-nginx-gateway-fabric.md" >}}) and save the public IP
+  address and port of NGINX Gateway Fabric into shell variables:
+
+   ```text
+   GW_IP=XXX.YYY.ZZZ.III
+   GW_PORT=<port number>
+   ```
+
+{{< note >}}In a production environment, you should have a DNS record for the external IP address that is exposed, and it should refer to the hostname that the gateway will forward for.{{< /note >}}
+
+
+## Response Header Filter
+
+In this guide we will deploy NGINX Gateway Fabric and use HTTPRoute resources to route traffic to the server. We will use `ResponseHeaderModifier` filter to modify the HTTP response headers.
+
+
+### Deploy the Headers application
+
+Begin by deploying the `headers` applications:
+
+   ```shell
+   kubectl apply -f https://raw.githubusercontent.com/nginxinc/nginx-gateway-fabric/tree/main/examples/http-response-header-filter/headers.yaml
+   ```
```

Comment:
what is the best practice around providing links? we use release tags in the previous guides, in that case should this point to `1.3` since its aimed for that

Reply:
I think it should be fine to use 1.3. Before our next release, we'll submit a PR to update all 1.2 strings to 1.3 anyway. The docs in the `main` branch are not published and therefore are subject to be "broken" as they are updated during development.